extractor/src/main/java/org/schabi/newpipe/extractor/services/media_ccc/extractors/MediaCCCStreamExtractor.java

```diff
@@ -170,15 +170,13 @@ public String getHlsUrl() throws ParsingException {
         return null;
     }
 
-    @Nonnull
     @Override
     public List<SubtitlesStream> getSubtitlesDefault() throws IOException, ExtractionException {
         return null;
```

Comment:
As far as I know this is not good practice, an empty `List` should be returned instead. Though I am not sure if it is normal to see null instead of empty lists in NewPipeExtractor. Same 5 lines below